archive-press_quote.php

```diff
@@ -0,0 +1,71 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
+    <div class="container">
+
+<?php
+if( have_posts() ) {
+  $i = 0;
+
+  while( have_posts() ) {
+    the_post();
+    $quote = get_post_meta($post->ID, '_igv_quote_text', true);
+    $link = get_post_meta($post->ID, '_igv_quote_link', true);
+
+    $title = get_the_title();
```

Comment:
i'm not sure the title is the right place to keep the name of the publication. what if they get multiple mentions in the same publication, then you would multiple press posts with the title "New York Times" for example. Perhaps it is better that the use the Title field however it best suits them, and we make a metabox for the publication name?